qemu/tests/cfg/convert_after_resize_snapshot.cfg

```diff
@@ -0,0 +1,24 @@
+- qemu_img_convert:
+    only qcow2
+    virt_test_type = qemu
+    type = convert_after_resize_snapshot
+    kill_vm = yes
+    start_vm = no
+    force_create_image = no
+    create_image = yes
+    remove_image = yes
+    images = "base"
```

Comment:
if the image tag is not `image1` and in `.py` QemuImg.create() is not explicit called,  `force_create_image_base = yes` is required to create a new image for test in the loop, otherwise it will use  `images/base` created by other test, which is not this case indented to do.